### PR TITLE
Fix dependencies

### PR DESCRIPTION
**Pull request by WindzCUHK**
# Description

## Changes

- copy dependencies from [yahoo/k8s-athenz-identity](https://github.com/yahoo/k8s-athenz-identity) to `third_party/*`
- update `go.mod`
- add license header
  - check license header in workflow

## Type of change

- [ ] Bug fix
- [ ] New feature
- [x] Refactoring (no functional changes, no api changes)
- [ ] Non-code changes (update documentation, pipeline, etc.)

### Flags

- [ ] Breaks backward compatibility
- [ ] Requires a documentation update
- [ ] Has untestable code

### CODE REVIEW

File: third_party/log/log.go [hunk 1]
```diff
@@ -0,0 +1,163 @@
+// Copyright 2020, Verizon Media Inc.
```
Comment (mlajkim) [inline third_party/log/log.go:1]: @WindzCUHK  Do you think we should mention that we got these code from the following link?
https://github.com/yahoo/k8s-athenz-identity/tree/b0ce39fad8332f6fe2d891df6d37ed94de59f15f/pkg

Totally optional and if you think we dont need, I am good.

**Review by mlajkim — APPROVED** [review]
lgtm

File: third_party/log/log.go (hunk 1 above)
Comment (WindzCUHK) [reply third_party/log/log.go:1]: thx, added in: [e23d7ce](https://github.com/WindzCUHK/k8s-athenz-sia/pull/6/commits/e23d7cea29dd8270c56a78df4a450e172958d9be)

Comment (WindzCUHK) [reply third_party/log/log.go:1]: @mlajkim  Please help to merge the PR to the main repo.
https://github.com/AthenZ/k8s-athenz-sia/pull/19

**Pull request closed without merging**